sdks/python/container/boot.go

```diff
@@ -210,15 +224,29 @@ func main() {
 	wg.Add(len(workerIds))
 	for _, workerId := range workerIds {
 		go func(workerId string) {
+			defer wg.Done()
 			log.Printf("Executing: python %v", strings.Join(args, " "))
-			log.Fatalf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
-			wg.Done()
+			log.Printf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
 		}(workerId)
 	}
 	wg.Wait()
 }
 
-// setup wheel specs according to installed python version
+// setupVenv initialize a local Python venv and set the corresponding env variables
+func setupVenv(dir string) error {
+	log.Printf("Initializing temporary Python venv ...")
+	if _, err := os.Stat(filepath.Join(dir, "pyenv.cfg")); os.IsNotExist(err) {
```

Comment:
I think you meant pyvenv.cfg

Reply:
Good catch. I guess I'm gonna remove this part anyway now that I'm binding the venv to the worker.